src/hotspot/share/memory/metaspace/counters.hpp

```diff
@@ -43,7 +44,7 @@ class AbstractCounter {
   T _c;
 
   // Only allow unsigned values for now
-  STATIC_ASSERT(IsSigned<T>::value == false);
+  STATIC_ASSERT(std::is_signed<T>::value == false);
```

Comment:
[pre-existing] I was going to say that we use `! _bool-expr_` rather than
`_bool-expr_ == false`, and I thought that was in the Style Guide.  But it
seems I was wrong about the Style Guide, and there are a surprising (to me)
number of occurrences of this sort of thing in our code (> 200).